beetsplug/smartplaylist.py

```diff
@@ -210,6 +220,8 @@ def update_playlists(self, lib):
                 for path in m3us[m3u]:
                     if self.config['forward_slash'].get():
                         path = path_as_posix(path)
-                    f.write(path + b'\n')
+                    if self.config['urlencoded'].get():
+                        path = bytestring_path(pathname2url(path))
```

Comment:
Again, I'm not sure `bytestring_path` is the right thing here…

Reply:
@sampsyo without it CI fails ;)

Let me show it as separated commit